config/settings.yml

```diff
@@ -131,21 +131,6 @@
     :purge_window_size: 10000
 :embedded_ansible:
   :job_data_retention_days: 120
```

Comment:
This can go also @NickLaMuro I thought we had that conversation somewhere in here ... looks like it got lost.

Reply:
Oh, nope, the conversation is just in the comment below for some reason.

Reply:
This is the comment being referenced:

https://github.com/ManageIQ/manageiq/pull/18687#discussion_r290526456

While I am cleaning up other bits, can you can confirm if this was decided on for being deleted?  It was unclear in the linked comment thread above.

Reply:
Yeah, this can go.

The runner utility will clean up the filesystem after we grab the results so there will be no "job data" left around.

If we want to set a retention period for ansible job database objects, then we should probably do it the same way it's done elsewhere (take [reporting](https://github.com/ManageIQ/manageiq/blob/01e077e98ff0a5029ad969861bf037299cebf540/config/settings.yml#L999-L1002) for example).

Reply:
Resolving, but of note, I ended up just removing the entire `:embedded_ansible:` section as a result since this was the item under that category.

Shouldn't be a problem, though possibly there are some tests relying on it that weren't an issue before (we will see).

Also, if we do decided we want to add some settings, we will just have to add it back at that time.